tests/data/placebo/test_assumed_session_session_policy/sts.AssumeRole_1.json

```diff
@@ -0,0 +1,22 @@
+{
+    "status_code": 200,
+    "data": {
+        "Credentials": {
```

Comment:
I had to trim down some attributes in this file because my laptop would not allow those to be sent (even the fake ones). However, I'm in the process of getting approval so that I can send the file as expected. The test failure will get fixed